frontend/Gruntfile.js

```diff
@@ -114,6 +119,18 @@ module.exports = function (grunt) {
                     preserveLicenseComments: false,
                     out: '<%= dirs.publicDir.javascripts %>/tools.js'
                 }
+            },
+            compileAdmin: {
+                options: {
+                    name: 'src/modules/admin/admin',
```

Comment:
It would be nice if we could separate the admin JS out too, so not part of our core app modules. Could just mean putting stuff in `src/admin` rather than in `src/modules/admin`.